subprojects/common/core/src/main/java/hu/bme/mit/theta/core/utils/ExprSimplifier.java

```diff
@@ -1175,6 +1182,29 @@ private Expr<BoolType> simplifyIntLt(final IntLtExpr expr, final Valuation val)
         return expr.with(leftOp, rightOp);
     }
 
+    /*
+     * Enums
+     */
+
+    private Expr<BoolType> simplifyEnumEqExpr(final EnumEqExpr expr, final Valuation val) {
```

Comment:
Should also be implemented for EnumNeqExprs

Reply:
Fixed